src/promptflow-devkit/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Bugs Fixed
 - Fix the issue that import error will be raised after downgrading promptflow from >=1.10.0 to <1.8.0.
+- Fix the issue that `pf flow serve` is broken with exception `NotADirectoryError`.
```

Comment:
should add the title #v1.10.1, here is v1.11.0, but you could update this on the release branch

Reply:
yes, I planed to update this on release branch - or it will be strange if we have both upcoming 1.11.0 and 1.10.1